catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
```

Comment:
It would be nice if this could be in the `config/` directory or `.github/` instead of adding another root-level file.